ClickEvent is not defined in Typescript types

Use of undefined `ClickEvent` breaks my typescript build. `ClickEvent` is not defined in the Typescript DOM library (`MouseEvent` used elsewhere in this file is). 

https://github.com/tscanlin/tocbot/blob/c56342fb1b32f11b4c5e6e2f8bfc7339aae254cf/index.d.ts#L93

```sh
../../node_modules/tocbot/index.d.ts:93:19 - error TS2304: Cannot find name 'ClickEvent'.

93     onClick?: (e: ClickEvent) => void;
                     ~~~~~~~~~~
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -90,7 +90,7 @@ declare namespace tocbot {
     // onclick function to apply to all links in toc. will be called with
     // the event as the first parameter; and this can be used to stop;
     // propagation; prevent default or perform action
-    onClick?: (e: ClickEvent) => void;
+    onClick?: (e: MouseEvent) => void;
 
     // orderedList can be set to false to generate unordered lists (ul)
     // instead of ordered lists (ol)

